client/src/components/molecules/OngoingAgendaCard.tsx

```diff
@@ -0,0 +1,81 @@
+import React, { useState } from "react";
+import { Box, Card, Text, Button } from "@/components/atoms";
+import { OngoingAgenda } from "biseo-interface/agenda";
+import {
+  ChoiceComponent,
+  CompletedChoice,
+  NotVotableChoice,
+} from "@/components/molecules";
+import { useAgenda } from "@/services/agenda";
+
+type OngoingAgendaProps = {
+  agenda: OngoingAgenda;
+};
+
+export const OngoingAgendaCard: React.FC<OngoingAgendaProps> = ({ agenda }) => {
+  const [chosenChoiceId, setChosenChoiceId] = useState(0);
+  const { voteAgenda } = useAgenda(state => ({
+    voteAgenda: state.voteAgenda,
+  }));
+
+  const vote = () => {
+    if (chosenChoiceId === 0) {
+      return null;
+    } else {
+      voteAgenda(chosenChoiceId, agenda.id);
+    }
+  };
+
+  const choose = (choiceId: number) => {
+    if (choiceId === chosenChoiceId) {
+      setChosenChoiceId(0);
+    } else {
+      setChosenChoiceId(choiceId);
+    }
+  };
+
+  const choices = agenda.user.votable ? (
+    agenda.user.voted ? (
+      <CompletedChoice />
+    ) : (
+      agenda.choices.map((choice, id) => (
+        <ChoiceComponent
+          key={id}
+          choice={choice}
+          chosen={choice.id == chosenChoiceId}
+          onClick={() => choose(choice.id)}
+        ></ChoiceComponent>
+      ))
+    )
+  ) : (
+    <NotVotableChoice />
+  );
+
+  return (
+    <Card primary>
+      <Box dir="column" gap={10}>
+        <Box>
+          <Text variant="title2" color="black">
+            {agenda.title}
+          </Text>
+          <Text variant="subtitle" color="gray500">
+            {agenda.content}
+          </Text>
+        </Box>
+        <Box dir="column" gap={6}>
+          <Text variant="body" color="blue600">
+            {agenda.resolution}
+          </Text>
+          {choices}
+        </Box>
+        <Box dir="row" justify="end" w="fill">
+          <Button onClick={() => vote()}>
+            <Text variant="option1" color="blue600">
+              투표하기
+            </Text>
+          </Button>
```

Comment:
이 버튼은 옵션이 선택되지 않은 경우에는 비활성화 되도록 하는 것이 User Experience 상으로 좋을 것 같아요!
```suggestion
          <Button onClick={() => vote()} disabled={chosenChoiceId === 0}>
            <Text variant="option1" color="blue600">
              투표하기
            </Text>
          </Button>
```

추가적으로 `Button` 컴포넌트에 대한 disabled 스타일이 추가되어야 할 것 같네요(디자인이 있는지는 확인해보고 없으면 chillo님께 요청해야 할 것 같아요).

Reply:
옹 시간 절약을 위해 그냥 카톡으로 바로 물어봐도 될 것 같긴 해용